BlueShift-iOS-SDK/BlueShiftNotificationConstants.h

```diff
@@ -33,7 +33,7 @@
 #define kNotificationCategoryViewCartIdentifier                         @"view_cart"
 #define kNotificationActionOpenCartIdentifier                           @"open_cart"
 
-#define kNotificationCategorySilentPushIdentifier                       @"silent_push"
+#define kNotificationCategorySilentPushIdentifier                       @"silent push"
```

Comment:
does this have something to do with keys from payload?

Reply:
yes. This key is coming from the push payload for silent push template type.